src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/Symbols/EENamedTypeSymbol.cs

```diff
@@ -349,6 +349,7 @@ internal override bool IsInterface
         internal sealed override NamedTypeSymbol NativeIntegerUnderlyingType => null;
 
         internal override bool IsRecord => false;
+        internal override bool HasPossibleWellKnownCloneMethod() => false;
```

Comment:
How are records represented in the EE? This line caught me a bit off guard. Must be I don't have the full pictuer here.

Reply:
I'm not sure how records are represented in EE, but this doesn't particularly change whatever it is. @tmat, does EE have records support? If so, we likely need to file a separate bug and make a change here.`